library/src/main/java/com/google/android/material/motion/runtime/NamedPlan.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2016 The Material Motion Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.android.material.motion.runtime;
+
+/**
+ * Similar to {@link Plan} in that a named plan is an object representing what you want something to do.
```

Comment:
"Similar to" is not completely correct, since NamedPlan is-a Plan. I would also remove much of the explanation of how NamedPlan and Plan act in the same way. That would reduce the javadoc down to:

```
/**
 * A named plan is a {@link Plan} whose performer supports adding and remove the plan by name. Register a named plan by calling {@link Scheduler#addNamedPlan}, and remove it by calling {@link Scheduler#removeNamedPlan}. 
 *
 * A named plan or family of named plans enables fine configuration of a performer's behavior.
 *
 * @see <a>spec</a> 
 */
```